recipes/default.rb

```diff
@@ -500,67 +519,21 @@ class ::Chef::Recipe
 include_recipe 'firewall'
 include_recipe 'selinux::permissive'
 include_recipe 'yum-qemu-ev'
+include_recipe 'build-essential'
 include_recipe 'openstack-common'
 include_recipe 'openstack-common::logging'
 include_recipe 'openstack-common::sysctl'
+include_recipe 'openstack-common::python'
 include_recipe 'openstack-identity::openrc'
-include_recipe 'build-essential'
-
-execute 'uninstall >= fog-openstack-0.2.0' do
-  command "/opt/chef/embedded/bin/gem uninstall -v '>= 0.2.0' fog-openstack --no-user-install"
-  only_if "/opt/chef/embedded/bin/gem list -i -v '>= 0.2.0' fog-openstack"
-end
-
-# TODO: Replace this with openstack-common::client when switching to Pike
-python_runtime '2' do
-  provider :system
-  pip_version '9.0.3'
-end
-
-python_virtualenv '/opt/osc' do
-  system_site_packages true
-end
-
-python_package 'cliff' do
-  version '2.9.0'
-  virtualenv '/opt/osc'
-end
-
-python_package 'os_client_config' do
-  version '1.28.0'
-  virtualenv '/opt/osc'
-end
-
-python_package 'osc_lib' do
-  version '1.7.0'
-  virtualenv '/opt/osc'
-end
-
-python_package 'openstacksdk' do
-  version '0.9.18'
-  virtualenv '/opt/osc'
-end
-
-python_package 'dogpile.cache' do
-  version '0.6.8'
-  virtualenv '/opt/osc'
-end
-
-python_package 'python-openstackclient' do
-  version node['openstack']['common']['client_version']
-  virtualenv '/opt/osc'
-end
+include_recipe 'openstack-common::client'
 
+# We're now using the packages openstack client
 link '/usr/local/bin/openstack' do
   to '/opt/osc/bin/openstack'
+  action :delete
 end
```

Comment:
Shouldn't you remove the `/usr/local/bin/openstack` binary? You won't be using the new binary otherwise AFAICT since the PATH on oprod-controller1.osuosl.org is `/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin`